### DISCUSSION TITLE
Norwegian keyboard missing backtick/grave only in some applications

### DISCUSSION THREAD

**Discussion by torpeanders** [regolith-linux/regolith-desktop#1048] [General]
I'm not sure if this is a Regolith issue, since I'm new to Wayland/Sway and just installed Ubuntu 24.04. Thought it worth posting here just in case it's an issue that should be fixed before 3.2

I'm running Regolith/Sway 3.2 Beta 4 on Ubuntu 24.04, and have a Norwegian keyboard layout. My issue is not present in the default Ubuntu Desktop.

To get backtick/grave on an NO keyboard you hit "Shift+<the-key-to-the-left-of-backspace". For me this generates the correct symbol in Emacs/gnome-terminal/Firefox, but nothing in Slack and Brave Browser. Could it be a Regolith issue. If not, do any of you have an idea where the bug might lie?

**Comment by flexd** [comment]
Fun times! I get this too. I just upgraded from Ubuntu 22.04 to 24.04, and now I have no grave/backtick '`' in certain applications. It's working fine here in Firefox, but in a Mattermost client, it's not. I guess maybe it's via XWayland perhaps? I'm not sure. I don't know how to see if a application is running natively in Wayland or using X.

Edit: Running xwininfo does not give any output at all on Firefox and my terminals (on Wayland), but if I click the Mattermost window it returns this
```
xwininfo

xwininfo: Please select the window about which you
          would like information by clicking the
          mouse in that window.

xwininfo: Window id: 0x400004 "Mattermost Desktop App"

  Absolute upper-left X:  1928
  Absolute upper-left Y:  8
  Relative upper-left X:  1928
  Relative upper-left Y:  8
  Width: 2547
  Height: 1399
  Depth: 24
  Visual: 0x4d
  Visual Class: TrueColor
  Border width: 0
  Class: InputOutput
  Colormap: 0x4c (installed)
  Bit Gravity State: NorthWestGravity
  Window Gravity State: NorthWestGravity
  Backing Store State: NotUseful
  Save Under State: no
  Map State: IsViewable
  Override Redirect State: no
  Corners:  +1928+8  -2565+8  -2565-33  +1928-33
  -geometry 2547x1399+1928+8
  ```

So I suppose that means that is running in X.

Edit 2: Also missing ~ and other symbols, so it's like the Xwayland session is defaulting to some strange keymap
